manage_breast_screening/notifications/management/commands/send_failed_message_batch.py

```diff
@@ -0,0 +1,43 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from manage_breast_screening.notifications.api_client import ApiClient
+from manage_breast_screening.notifications.management.commands import (
+    message_batch_functions,
+)
+from manage_breast_screening.notifications.models import MessageBatch
+
+
+class Command(BaseCommand):
+    """
+    Django Admin command which takes an ID of a MessageBatch with
+    a failed status and retries sending it to the Communications API.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument("message_batch_id")
+
+    def handle(self, *args, **options):
+        # Look for the MessageBatch and check validity
+        batch_id = options["message_batch_id"]
+        message_batch = MessageBatch.objects.filter(id=batch_id).first()
```

Comment:
```suggestion
        message_batch = MessageBatch.objects.filter(
            id=batch_id, status="failed"
        ).first()
```
would mean we don't need to catch this case later.

Reply:
ohh yeah good point!